zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/AdHocSubprocessActivityController.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.controller;
+
+import static io.camunda.zeebe.gateway.rest.RestErrorMapper.mapErrorToResponse;
+
+import io.camunda.service.AdHocSubprocessActivityServices;
+import io.camunda.service.AdHocSubprocessActivityServices.AdHocSubprocessActivity;
+import io.camunda.zeebe.gateway.protocol.rest.AdHocSubprocessActivityResult;
+import io.camunda.zeebe.gateway.protocol.rest.AdHocSubprocessActivitySearchQuery;
+import io.camunda.zeebe.gateway.protocol.rest.AdHocSubprocessActivitySearchQueryResult;
+import io.camunda.zeebe.gateway.rest.RequestMapper;
+import io.camunda.zeebe.gateway.rest.annotation.CamundaPostMapping;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@CamundaRestController
+@RequestMapping("/v2/ad-hoc-activities")
+public class AdHocSubprocessActivityController {
+
+  private final AdHocSubprocessActivityServices adHocSubprocessActivityServices;
+
+  public AdHocSubprocessActivityController(
+      final AdHocSubprocessActivityServices adHocSubprocessActivityServices) {
+    this.adHocSubprocessActivityServices = adHocSubprocessActivityServices;
+  }
+
+  @CamundaPostMapping(path = "/activatable")
+  public ResponseEntity<AdHocSubprocessActivitySearchQueryResult> findActivatableActivities(
```

Comment:
TODO add some validation to the request body. Not sure if we're able to use bean validation on the generated models, but otherwise we can implement a simple validation function (validating adHocSubprocessId not blank for example) on the controller level as other controllers already do.